src/views/SideMenu.vue

```diff
@@ -28,13 +28,19 @@
 
             <div class="columns is-gapless has-text-centered is-mobile menu-socials">
                 <div class="column">
-                    <img src="@/assets/img/social/twitter.png" alt="Twitter">
+                    <a href="https://twitter.com/camptocamporg">
+                        <img src="@/assets/img/social/twitter.png" alt="Twitter">
```

Comment:
can't we use SVG fonts instead?

Reply:
We do not have a social for google+ 

That said google+ is dead. Could we see with c2c assoce to remove it ?

Reply:
correction :  a svg font

Reply:
I agree we can remove google+. But my comment was more general: you are also using images for twitter and facebook.
There are definitely fonts for brands. See e.g. https://fontawesome.com/icons?d=gallery&s=brands

Reply:
yes, we still have it for twitter and fb, I will use them.

Reply:
FYI : https://forum.camptocamp.org/t/projet-ui-google/235265